bundler/lib/bundler/man/bundle-plugin.1.ronn

```diff
@@ -0,0 +1,61 @@
+bundle-plugin(1) -- Manage Bundler plugins
+==========================================
+
+## SYNOPSIS
+
+`bundle plugin` install PLUGINS [--source=<SOURCE>] [--version=<version>]
+                              [--git|--local_git=<git-url>] [--branch=<branch>] [--ref=<rev>]
+
+`bundle plugin` uninstall PLUGINS
+
+`bundle plugin` list
+
+`bundle plugin` help [COMMAND]
+
+## DESCRIPTION
+
+You can install, uninstall, and list plugin(s) with this command to extend functionalities of Bundler.
+
+## SUB-COMMANDS
+### install
+
+Install the plugin(s) from the source.
```

Comment:
Maybe just "Install the given plugin(s)"? When reading the examples below, I thought the one showing `--version` felt off because the header didn't mention it.

Reply:
Suggestion for the sentence was applied. Per `--version`, the first line in the header includes it, doesn't it?
```
$ MANWIDTH=48 MANPAGER=head exe/bundle plugin install --help
BUNDLE-PLUGIN(1)              BUNDLE-PLUGIN(1)

NAME
       bundle-plugin - Manage Bundler plugins

SYNOPSIS
       bundle     plugin    install    PLUGINS
       [--source=SOURCE]   [--version=version]
       [--git|--local_git=git-url]
       [--branch=branch] [--ref=rev]
```

Reply:
Yes, in the synopsis, I meant in the subcommand description.

Since it only talked about source ("Install the plugin(s) from the source."), but didn't mention "version", and then described different commands using "source related options", I was not sure if there was going to be a different section describing other options. That's why I suggested to make the description more generic.

Reply:
Actually it doesn't seem applied?

Reply:
Sorry, cb899acec274c48e5ad66f655d0963eb0a1d8778 was not actually pushed. Just did it.